lib/x11utils.pm

```diff
@@ -68,8 +69,9 @@ all possible options should be handled within loop to get unlocked desktop
 =cut
 sub ensure_unlocked_desktop {
     my $counter = 10;
+
     while ($counter--) {
-        my @tags = qw(displaymanager displaymanager-password-prompt generic-desktop screenlock screenlock-password authentication-required-user-settings authentication-required-modify-system);
+        my @tags = qw(displaymanager displaymanager-password-prompt generic-desktop screenlock screenlock-password authentication-required-user-settings authentication-required-modify-system guest-disabled-display);
         push(@tags, 'blackscreen') if get_var("DESKTOP") =~ /minimalx|xfce/;    # Only xscreensaver and xfce have a blackscreen as screenlock
```

Comment:
IIUC "blackscreen" and "guest-disabled-display" serve the same purpose. So far this really seems to be aarch64 specific so I suggest to only enable there, is it correct? Maybe it's because of running a more recent qemu on our aarch64 then of course enable it for all.

Reply:
I think this is more virtio specific.

Reply:
even better to use that as the condition

Reply:
@okurz I've seen the guest-disabled-display somewhere else in x86. I'd say is more of a generic thing, and only in the aarch specific case, I'd tag it as a possible response to the bug in question, as @ggardet is already confirming that this is not the bug he's seeing on opensuse